js/privatebin.js

```diff
@@ -4555,6 +4498,146 @@ jQuery.PrivateBin = (function($, RawDeflate) {
         return me;
     })();
 
+
+    /**
+     * initial (security) check
+     *
+     * @name   InitialCheck
+     * @param  {object} window
+     * @param  {object} document
+     * @class
+     */
+    var InitialCheck = (function () {
+        var me = {};
+
+        /**
+         * blacklist of UserAgents (parts) known to belong to a bot
+         *
+         * @private
+         * @enum   {Array}
+         * @readonly
+         */
+        const badBotUA = [
+            'Bot',
+            'bot'
+        ];
+
+        /**
+         * check if the connection is insecure
+         *
+         * @private
+         * @name   InitialCheck.isInsecureConnection
+         * @function
+         * @return {bool}
+         */
+        function isInsecureConnection()
+        {
+            // use .isSecureContext if available
+            if (window.isSecureContext === true || window.isSecureContext === false) {
+                return !window.isSecureContext;
+            }
+
+            const url = new URL(window.location);
+            // HTTP is obviously insecure
```

Comment:
So then, either revert the old commit by yourself or manually copy-paste/re-introduce the logic here.

Reply:
Will do, as said, but want to get to the bottom of what is showing up in those unit tests first.